araproc/analysis/standard_reco.py

```diff
@@ -599,3 +599,61 @@ def calculate_surface_corr_ratio(self, *maps, z_thresh=-10):
         
         return surf_corr_ratio, max_surf_corr_result, max_corr_result
 
+    def min_frac_corr_depth(self, corr_map, fraction=0.6, z_thresh=-10):
```

Comment:
I think this is another one we want to aggregate over maps, same as with the surface correlation ratio. Also if we have `z_thresh` let's make the default value `0`.

Reply:
Sorry I think we want this one to have a `*maps` arguments right?

Reply:
We do that in the "multiple" version: 'min_frac_corr_depth_multiple(self, *maps, fraction=0.6, z_thresh=0):' where each element of *maps is passed to 'min_frac_corr_depth(self, corr_map, fraction=0.6, z_thresh=-10)' and then identifies which map has minimum depth

Reply:
Whoops! Sorry I missed that. Awesome, then it's all set!